pkg/native/serializers/serializer_spdx23.go

```diff
@@ -353,3 +357,53 @@ func buildPackages(bom *sbom.Document) ([]*spdx.Package, error) { //nolint:unpar
 	}
 	return packages, nil
 }
+
+// ExtRefCategoryFromProtobomExtRef reads a protobom external reference struct and returns a
+// string with the corresponding category
+func (s *SPDX23) extRefCategoryFromProtobomExtRef(extref *sbom.ExternalReference) string {
+	switch extref.Type {
+	case sbom.ExternalReference_BOWER, sbom.ExternalReference_MAVEN_CENTRAL,
+		sbom.ExternalReference_NPM, sbom.ExternalReference_NUGET:
+		return spdx.CategoryPackageManager
+	case sbom.ExternalReference_SECURITY_ADVISORY, sbom.ExternalReference_SECURITY_FIX:
```

Comment:
what about adding sbom.ExternalReference_SWID here?

Reply:
Oh yeah. good catch. Can I push it in a follow-up PR to avoid rebase hell in the next PRs?

Reply:
Of course!